libs/blocks/section-metadata/section-metadata.js

```diff
@@ -95,5 +132,7 @@ export default async function init(el) {
   if (metadata.masonry) handleMasonry(metadata.masonry.text, section);
   if (metadata.delay) handleDelay(metadata.delay.text, section);
   if (metadata.anchor) handleAnchor(metadata.anchor.text, section);
+  // eslint-disable-next-line max-len
+  if (metadata['collapse-ups-mobile']) await handleCollapseSection(metadata['collapse-ups-mobile'].text, section);
```

Comment:
If `text` is only used to check for `on` value, I would say it is not necessary as a param 
```suggestion
  if (metadata['collapse-ups-mobile']?.text === 'on') await handleCollapseSection(section);
```